### PR TITLE
 docs(community): Add user story videos

### PR DESCRIPTION
**Pull request by drodio**

### CODE REVIEW

File: community/user-stories/index.md
Comment (dorbin) [inline community/user-stories/index.md:9]: s/us/Armory/

...just because I think users might get confused by a page referring to an "us" that is Armory instead of the whole Spinnaker OSS community.

File: community/user-stories/index.md
Comment (dorbin) [inline community/user-stories/index.md:79]: s/Thomas/Tomás/

Comment (dorbin) [reply community/user-stories/index.md:79]: Would it be possible to also change the spelling of his name on video?

File: community/user-stories/index.md
Comment (dorbin) [inline community/user-stories/index.md:8]: I would add `{% include toc %}` here, just so we see at the top that there are two sections--users/evaluators and builders/supporters.

**Review by dorbin — COMMENTED** [review]
LGTM, @drodio , pending review by @demobox .

Thanks!

File: community/user-stories/index.md
Comment (duftler) [inline community/user-stories/index.md:83]: s/Stephen/Steven

File: community/user-stories/index.md
Comment (duftler) [inline community/user-stories/index.md:87]: s/Philips/Phillips

**Review by demobox — CHANGES_REQUESTED** [review]
Do we have examples of similar content on other OSS project sites? I agree that user stories are powerful, but they also come close to marketing material. There's also prominent vendor logo placement in these videos, so I'd like to get @skim1420's thoughts on that, too.

I wonder if we could combine this with the ["Who's using Spinnaker"](https://www.spinnaker.io/community/#whos-using-spinnaker) section - that seems the most obvious place for this?

File: community/user-stories/index.md
Comment (robzienert) [inline community/user-stories/index.md]: `s/Robert/Rob` plz.

Comment (robzienert) [inline community/user-stories/index.md:75]:
```suggestion
###  Rob Zienert - Sr. Software Engineer - Netflix
```

File: community/user-stories/index.md
Comment (drodio) [reply community/user-stories/index.md:79]: @dorbin yes I'll get that fixed

File: community/user-stories/index.md
Comment (drodio) [reply community/user-stories/index.md:83]: I'll get that fixed

File: community/user-stories/index.md
Comment (drodio) [reply community/user-stories/index.md:87]: I'll get that fixed

File: community/user-stories/index.md
Comment (drodio) [reply community/user-stories/index.md:9]: I'll update that

**Comment by drodio** [comment]
@demobox --

@skim1420 had recommended creating a sibling page to Community/Misc/FAQs and @dorbin had recommended "As a sibling to /Community/Misc, but not to /Community/Misc/FAQ. So: /Community/User Stories/. And we could move (or cross-post) case studies there."

My hope is we can get a bunch more up, and evolve this page to make it more granular (for example, sections for companies just starting to use Spinnaker (and why they chose it) / those that are running it in production / those that have scaled it out in meaningful ways or have other pro-tips.  

So I do think it's better we have our own "user stories" section. But if you want me to move them over to the "who's using" section, just LMK.

File: community/user-stories/index.md
Comment (drodio) [reply community/user-stories/index.md:8]: @dorbin I originally had it on, but it covers up the first two videos: here's a screenshot: http://drod.io/8f610850ba89 and the youtube sizing on embeds doesn't work so I figured the simpler thing was to just remove the TOC. Open to suggestions, though.

**Comment by skim1420** [comment]
Sorry for steering you in the wrong direction; I hadn't looked at the videos myself prior to my suggestion. I'll defer to @dorbin and @demobox on these concerns, and on consideration I do understand @demobox 's thoughts here. 

For example, keeping content on the OSS site 3rd party neutral is important, and having a page effectively dedicated to interviews done by one group doesn't serve that.

On a tangent, perhaps a more regular body of persons who makes the calls on documentation matters makes sense, e.g. a SIG? I'll take that offline.

File: community/user-stories/index.md
Comment (dorbin) [reply community/user-stories/index.md:8]: @drodio I see what you mean. Consider having two bullet items here at the top, before even the intro paragraph, linking to the two h2s. But even that is minor. It's ok to leave out; I'm just concerned the other h2, way below, will get lost.

**Review by dorbin — COMMENTED** [review]
Good points, @skim, about a SIG docs and about neutrality of the OSS site. I defer to you and @demobox for this one.

**Comment by robzienert** [comment]
Closing. This is over a year old, so I doubt it'll get updated.

**Comment by dorbin** [comment]
@drodio Sorry this languished all this time. 
Not going to re-open this PR...yet.
* @deesprinter  has added some customer [success stories](https://www.spinnaker.io/success-stories/). I feel like that satisfies _some_ of your hope to "get a bunch more up," but it doesn't get these initial videos up.
* @demobox can you look at this again and we can revisit the discussion over this?

**Comment by dnilasor** [comment]
@dorbin, Hi, I just came onboard at Armory and had some ideas about how we could incorporate all of our existing video content onto the site. This project is active: https://github.com/nathancy/jekyll-embed-video and could probably be leveraged. I was thinking a div/area on the Success Stories page with a row or 2 of directly linked videos, and then a link from there below, like "More Videos" or something that redirects to a Videos page with a paginated grid of all the videos we want to add. 

I'm hoping to play around with this in the next several weeks and come up with an MVP that works, then submit a PR. I'd like to get these great video assets out there and linked directly to the project, polished or not. Thoughts?

**Comment by dnilasor** [comment]
@dorbin my idea behind doing it that way was that, the videos all fall in the category of "success stories" I think and with that design we aren't adding yet another thing to the cluttered nav bar. but idk what implications are for any vision of the site organization and am open to conversation. Will play with it asap to solidify my idea

**Comment by gcomstock** [comment]
Feel free to add this to the agenda for the documentation SIG on Wednesday!

https://tinyurl.com/yy2qxmyh

**Comment by dnilasor** [comment]
Okay, will do. Also, after looking at the code I can see that you already have video embeds set up and styled for use in the tutorials area so, even easier, can adopt that. yay : )

**Pull request closed without merging**
